README.md

```diff
@@ -312,44 +309,58 @@ metadata:
 spec:
   description: string # optional up to 1050 characters
   thresholdMetric: # either thresholdMetric or ratioMetric must be provided
-    metricSource:
-      metricSourceRef: string # optional, this field can be used to refer to DataSource object
-      type: string # optional, this field describes predefined metric source type e.g. Prometheus, Datadog, etc.
-      spec:
-        # arbitrary chosen fields for every data source type to make it comfortable to use
-        # anything that is valid YAML can be put here.
+    # either dataSourceRef or <<dataSourceName>> must be provided
+    dataSourceRef: string # refer to already defined DataSource object
+    <<dataSourceName>>: # inline whole DataSource e.g. cloudWatch, datadog, prometheus (arbitrary chosen, implementor decision)
+      # fields used for creating a connection with particular datasource e.g. AccessKeys, SecretKeys, etc.
+      # everything that is valid YAML can be put here
+    spec:
+      # arbitrary chosen fields for every DataSource type to make it comfortable to use (query)
```

Comment:
What do you mean by `(query)` in there? I think leaving it as `...type to make it comfortable to use` is clear enough description. Query isn't always case for data source, I can think of data source type that can send/stream SLIs into our SLO.

Reply:
Yeah, you're right. I could write
> (e.g. query)
but now I think it's redundant so I'm getting rid of it